package.json

```diff
@@ -148,5 +148,6 @@
     "android": {
       "javaPackageName": "com.swmansion.rnscreens"
     }
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
Let me do that by myself, since we're quite in a rush today 😄

Reply:
Resolved in https://github.com/software-mansion/react-native-screens/pull/2077/commits/a7433f372c0c24259f0d11a67fc6f6273d4df5d6.

Reply:
Absolutely! I used 1.22.19 because the website showed it as the latest stable version (https://classic.yarnpkg.com/lang/en/) but looks like that's outdated.